ngraph/frontend/onnx/frontend/CMakeLists.txt

```diff
@@ -44,7 +44,9 @@ if(COMMAND ie_add_vs_version_file)
                            FILEDESCRIPTION "nGraph ONNX frontend library")
 endif()
 
-target_link_libraries(onnx_ngraph_frontend PUBLIC ngraph PRIVATE frontend_manager ngraph::builder onnx_common inference_engine_transformations)
+target_link_libraries(onnx_ngraph_frontend PUBLIC ngraph PRIVATE frontend_manager ngraph::builder onnx_common
+                                                                 inference_engine_transformations
+                                                                 offline_transformations)
```

Comment:
do we still need `offline_transformations`?

Reply:
removed.